GVFS/GVFS.Common/GitHubUpgrader.cs

```diff
@@ -459,7 +478,11 @@ private bool TryRunInstallerForAsset(string assetId, out int installerExitCode,
             {
                 string logFilePath = GVFSEnlistment.GetNewLogFileName(ProductUpgraderInfo.GetLogDirectoryPath(), Path.GetFileNameWithoutExtension(path));
                 string args = installerArgs + " /Log=" + logFilePath;
-                this.RunInstaller(path, args, out installerExitCode, out error);
+                string gvfsCN = "Microsoft Corporation";
+                string gitCN = "Johannes Schindelin";
+                string certCN = assetId.Equals(GVFSAssetId) ? gvfsCN : assetId.Equals(GitAssetId) ? gitCN : null;

```

Comment:
Please don't next conditional operators. You can use a `switch` statement with strings.